std_msgs/Int32MultiArray grid
---
std_msgs/Int32MultiArray altered_grid
